README.md

```diff
@@ -184,6 +184,34 @@ cache_fragment(post)
 cache_fragment(post) { post }
 ```
 
+Using literals: Even when using a same string for all queries, the cache changes per argument because of the query_key.
+
+```ruby
+def post(id:)
+  cache_fragment('find_post') { Post.find(id) }
+end
+```
+
+Combining with options:
+
+```ruby
+def post(id:)
+  cache_fragment('find_post', expires_in: 5.minutes) { Post.find(id) }
+end
+```
+
+Dynamic cache key:
+
+```ruby
+def post(id:)
+  last_updated_at = Post.select(:updated_at).order(updated_at: :desc).first&.updated_at
```

Comment:
Not sure I understand the use case: the goal is to keep all posts cached until any of posts is changed, and, when it happens, to expire the cache, am I right?

Reply:
Yes!  Perhaps it was a bad example from my side because in my case I was caching a field that was not based on the specific ID, it was based on timing and other properties. Because of that, in my field, I was expiring based and safer on any change for all records.

So in this Post's example, I think it is better to have the specific updated_at column. 

```suggestion
  last_updated_at = Post.select(:updated_at).find_by(id: id)&.updated_at
```

Note here that I used the select to make the query as fastest and light as possible.